.github/workflows/ci-aot.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false  # ensures the entire test matrix is run, even if one permutation fails
       matrix:
-        os: [ windows-latest ]
+        os: [ windows-latest, ubuntu-latest ]
```

Comment:
@vitek-karas Would you recommend we test this on both Windows and Linux? Is there any value in doing that?

Reply:
I don't think there's much value right now in running this on both platforms. I would probably go with just Linux (it's cheaper to run).

Later on if/when we add more "meat" to the AOT test app (so that it actually runs and exercises some interesting code) it might make sense to test on both platforms.